kclvm/sema/src/advanced_resolver/node.rs

```diff
@@ -1300,13 +1300,6 @@ impl<'ctx> AdvancedResolver<'ctx> {
                     .get(&self.ctx.get_node_key(&arg.id))
                 {
                     match arg_ref.get_kind() {
-                        crate::core::symbol::SymbolKind::Value => {
```

Comment:
For value symbol, this will get the definition symbol instead of ref symbol(for a, actually it will get symbol in line 1 instead of line 29). This problem may be related to the function of find_ref. I will deal with it later.